app/presenters/publishing_api/html_attachment_presenter.rb

```diff
@@ -38,7 +38,7 @@ def links
     def edition_links
       {
         parent: parent_content_ids, # please use the breadcrumb component when migrating document_type to government-frontend
-        organisations: parent.organisations.pluck(:content_id).uniq,
+        organisations: (parent.lead_organisations.pluck(:content_id) + parent.supporting_organisations.pluck(:content_id)).uniq,
```

Comment:
Looks like there's already an [existing method](https://github.com/alphagov/whitehall/blob/3f7a05fd6959784db3463febe977ccc26ee05cec/app/presenters/publishing_api/html_attachment_presenter.rb#L95) for retrieving the `lead_org_id` (and it uses `.try` to fail gracefully). Can you reuse that method, and add an equivalent for `supporting_org_ids` that uses `.try` in the same way?

Reply:
Ah, just spotted the existing method only returns one lead org ID, you probably want them all. But `lead_org_ids` seems a fair method to add.